examples/FCCee/bsm/LLPs/DisplacedHNL/analysis_final.py

```diff
@@ -170,7 +170,7 @@
     "Reco_DecayVertex_y_prompt":    {"name":"RecoDecayVertex.position.y",  "title":"Reco decay vertex y [mm]",            "bin":100,"xmin":-0.01 ,"xmax":0.01},
     "Reco_DecayVertex_z_prompt":    {"name":"RecoDecayVertex.position.z",  "title":"Reco decay vertex z [mm]",            "bin":100,"xmin":-0.01 ,"xmax":0.01},
     "Reco_DecayVertex_chi2":        {"name":"RecoDecayVertex.chi2",        "title":"Reco decay vertex #chi^{2}",          "bin":100,"xmin":0 ,"xmax":3},
-    "Reco_DecayVertex_probability": {"name":"RecoDecayVertex.probability", "title":"Reco decay vertex probability",       "bin":100,"xmin":0 ,"xmax":10},
+    "Reco_DecayVertex_ndf":         {"name":"RecoDecayVertex.ndf",         "title":"Reco decay vertex fit ndf",           "bin":50,"xmin":0 ,"xmax":50},
```

Comment:
I think in this case, it would be better to calculate the `Reco_DecayVertex_probability` upfront from `chi2` and `ndf`, since there is a clear expectation on how the probability distribution should look like, but the `ndf` distribution is probably not really meaningful.

I guess in this case that definition would have to go into a previous step. Is there a common definitions step, where we could put this? @kjvbrt can you tell us where to put this, or ping someone who knows?

Reply:
I think if we add something like:
```python
                .Define("Reco_DecayVertex_probability",  "TMath::Prob(RecoDecayVertex.chi2, RecoDecayVertex.ndf)")
```
here
https://github.com/HEP-FCC/FCCAnalyses/blob/c8cbcf0438c15bca1d2be1c34ce688b61b468d92/examples/FCCee/bsm/LLPs/DisplacedHNL/analysis_stage1.py#L199-L200

the changes here would amount to changing the `name` in the nested dictionary, and we would not change what the code actually does.